src/Dapr.PluggableComponents/Components/SecretStores/SecretStoreGetRequest.cs

```diff
@@ -0,0 +1,37 @@
+﻿// ------------------------------------------------------------------------
+// Copyright 2023 The Dapr Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+using Dapr.Proto.Components.V1;
+
+namespace Dapr.PluggableComponents.Components.SecretStores;
+
+/// <summary>
+/// Represents properties associated with a request to retrieve a secret from a secret store.
+/// </summary>
+/// <param name="secretName">The secret that should be retrieved.</param>
+public sealed record SecretStoreGetRequest(string secretName)
```

Comment:
For `record` types, arguments passed in the constructor are mapped to read-only properties, so their names should conform to property naming guidelines (e.g. `SecretName` rather than `secretName`).